_includes/scenarios/integrate/share-own-attribute-to-peer.md

```diff
@@ -1,11 +1,285 @@
+
+<!--- TODO: Insert link to "Create own IdentityAttribute" guide --->
+
+```jsonc
+{
+  "@type": "LocalAttribute",
+  "id": "<ID of LocalAttribute>",
+  "createdAt": "<creation date of LocalAttribute>",
+  "content": {
+    "@type": "IdentityAttribute",
+    "owner": "<Address of Sender>",
+    "value": {
+      "@type": "BirthDate",
+      "day": <day of birth date>,
+      "month": <month of birth date>,
+      "year": <year of birth date>
+    }
+  }
+}
+```
+
+In our example, the Sender wants to share the IdentityAttribute with the Recipient. To do so, it needs to insert the `id` of the corresponding LocalAttribute into the `sourceAttributeId` property and the IdentityAttribute itself into the `attribute` property of the [ShareAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#shareattributerequestitem) contained within the `items` property of the [Request]({% link _docs_integrate/data-model-overview.md %}#request) for sharing Attributes. The value of the `mustBeAccepted` property of the ShareAttributeRequestItem is set to `true` in this example.
+
+```jsonc
+{
+  "@type": "Request",
+  "items": [
+    {
+      "@type": "ShareAttributeRequestItem",
+      "mustBeAccepted": true,
+      "attribute": {
+        "@type": "IdentityAttribute",
+        "owner": "<Address of Sender>",
+        "value": {
+          "@type": "BirthDate",
+          "day": <day of birth date>,
+          "month": <month of birth date>,
+          "year": <year of birth date>
+        }
+      },
+      "sourceAttributeId": "<ID of LocalAttribute>"
+    }
+  ]
+}
+```
+
+### Example of sharing an own RelationshipAttribute
+
+We now consider the case in which the Sender has an active [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) with a third party and owns a RelationshipAttribute of type [ProprietaryString]({% link _docs_integrate/attribute-values.md %}#proprietarystring) of this Relationship. The Sender can request to share this RelationshipAttribute with the Recipient if its `confidentiality` is `"protected"` or `"public"`. In our example, we assume that the `confidentiality` of the RelationshipAttribute is `"public"` and that it is stored locally within the `content` property of a corresponding [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) with a [LocalAttributeShareInfo]({% link _docs_integrate/data-model-overview.md %}#localattributeshareinfo) of the Sender.
+
+```jsonc
+{
+  "@type": "LocalAttribute",
+  "id": "<ID of LocalAttribute>",
+  "createdAt": "<creation date of LocalAttribute>",
+  "content": {
+    "@type": "RelationshipAttribute",
+    "owner": "<Address of Sender>",
+    "key": "<key of RelationshipAttribute>",
+    "confidentiality": "public",
+    "value": {
+      "@type": "ProprietaryString",
+      "title": "<title of RelationshipAttribute>",
+      "value": "<actual value of RelationshipAttribute>"
+    }
+  },
+  "shareInfo": {
```

Comment:
Realistically, the shareInfo should also have a defined `requestReference` with the `id` of the Request that was used to create the RelationshipAttribute for the third party.

Reply:
As I understand it, the shareInfo can contain either a `requestReference` or a `notificationReference`. As it is not known whether the Attribute has been sent via a Request or a Notification and this information does not really play a role, I have omitted it for the time being. What do you think about this?